mlflow/store/file_store.py

```diff
@@ -190,10 +190,9 @@ def list_experiments(self, view_type=ViewType.ACTIVE_ONLY):
         return experiments
 
     def _create_experiment_with_id(self, name, experiment_id, artifact_uri):
+        assert artifact_uri, "experiment artifact URI must be a non-empty string"
```

Comment:
This `assert` is in production path.
If you intend this to be a check, can you if an `if not artifact_...` and raise an `MlflowException`

Reply:
Done! Thanks.